Makefile.inc

```diff
@@ -54,6 +54,11 @@ skipclippy:
 
 # Testing. A lot of directories can't test either, yet.
 test:
+	cargo test
 
 skiptest:
 	echo Not doing test for this target
+
+coverage:
+	cargo tarpaulin --out Xml --output-dir ./coverall
+	curl -s "https://codecov.io/bash" | bash -s
```

Comment:
remove the curl here, that's covered by the GitHub Action and does not work anywhere else

Reply:
I wanted to allow people to submit coverage from their own computer. Do I do another make recipe?

Reply:
I wanted to allow people to submit coverage from their own computer. Do I do another make recipe?

Reply:
When running the command locally, you also get the reports locally. However, due to access rights, there is no chance for arbitrary users pushing coverage to arbitrary projects. That is what the access controls implement between GitHub and other services. Otherwise, anyone could interfer with any project. Instead, if you wanted to publish your own, you could do so from your own fork and set up your own CI services for it, run your own infra, etc..